core/deployment/src/main/java/io/quarkus/deployment/configuration/RunTimeConfigurationGenerator.java

```diff
@@ -783,6 +736,34 @@ public void run() {
             generateDefaultValuesConfigSourceClass(buildTimeRunTimePatternMap, BTRTDVCS_CLASS_NAME);
         }
 
+        private static void configSweepLoop(MethodDescriptor siParserBody, MethodCreator clinit2, ResultHandle clinitConfig2) {
```

Comment:
Minor nitpick: The arguments names all need to be made more general I think :)

Reply:
Oops, this was an intellij automatic refactoring and I did not look at the method names.